target/linux/ramips/mt7621/config-6.6

```diff
@@ -115,6 +115,8 @@ CONFIG_IRQ_DOMAIN_HIERARCHY=y
 CONFIG_IRQ_FORCED_THREADING=y
 CONFIG_IRQ_MIPS_CPU=y
 CONFIG_IRQ_WORK=y
+CONFIG_JFFS2_LZO=y
+CONFIG_JFFS2_ZLIB=y
```

Comment:
Are these two compressions needed? You set `--compression-mode=none` when creating the jffs2.

Reply:
These are needed, since the factory uboot expects compression, and we have to store the kernel in the JFFS2 partition for uboot to boot it.

Reply:
Are both of them needed?